src/js/i18n/en-US.yml

```diff
@@ -1,5 +1,40 @@
 ---
 careOptsFrontend:
+  admin:
```

Comment:
good, was toying around with doing this myself. I'll have conflicts, but it needed to happen